### PR TITLE
update zh-cn translations for v3.12.1

### PR DESCRIPTION
**Pull request by mercury233**

### CODE REVIEW

File: src/translations/sqlb_zh.ts [hunk 1]
```diff
@@ -214,67 +214,68 @@
...
         <location filename="../Application.cpp" line="97"/>
         <source>  -q, --quit          Exit application after running scripts</source>
-        <translation type="unfinished"></translation>
+        <translation>  -q, --quit          在运行脚本后退出应用程序</translation>
     </message>
     <message>
         <location filename="../Application.cpp" line="98"/>
         <source>  -s, --sql &lt;file&gt;    Execute this SQL file after opening the DB</source>
-        <translation type="unfinished"></translation>
+        <translation>  -s, --sql [文件]    在打开数据库后执行此 SQL 文件</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_zh.ts:233]: Just curious, Why did you use [brackets] instead of the <> symbols?

File: src/translations/sqlb_zh.ts [hunk 2]
```diff
@@ -4864,32 +4865,32 @@ If you answer no we will attempt to import the data in the SQL file to the curre
...
         <location filename="../MainWindow.cpp" line="336"/>
         <source>Simplify Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>精简窗口布局</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="337"/>
         <source>Shift+Alt+0</source>
-        <translation type="unfinished"></translation>
+        <translation>Shift+Alt+0</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_zh.ts:4878]: We are asking to contributors not to translate shortcuts on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts).

There was a discussion about this in PR #2406 some time ago, but in that case,
the shortcut didn't work and we wasn't sure if that was a Qt bug, so we proceeded the merge without deletion.

If possible, delete the shortcut translation, and if the shortcut doesn't work after deleting,
please tell us so we can just merge it without deletion.

File: src/translations/sqlb_zh.ts [hunk 3]
```diff
@@ -6710,137 +6711,137 @@ Create a backup!</source>
...
         <location filename="../RemoteDock.ui" line="411"/>
         <source>Reload all data and update the views</source>
-        <translation type="unfinished"></translation>
+        <translation>重新载入所有数据并更新视图</translation>
     </message>
     <message>
         <location filename="../RemoteDock.ui" line="414"/>
         <source>F5</source>
-        <translation type="unfinished"></translation>
+        <translation>F5</translation>
```
Comment (lucydodo) [inline src/translations/sqlb_zh.ts:6789]: Here too :wink:

**Review by lucydodo — CHANGES_REQUESTED** [review]
Thanks for contributing!
Needed some modifications, so I make a review. Please check them. :smile:

File: src/translations/sqlb_zh.ts (hunk 1 above)
Comment (mercury233) [reply src/translations/sqlb_zh.ts]: Sorry I didn't notice the original text didn't use the brackets

Comment (lucydodo) [reply src/translations/sqlb_zh.ts]: Fine. Not much of a problem. :smile:

File: src/translations/sqlb_zh.ts (hunk 2 above)
Comment (mercury233) [reply src/translations/sqlb_zh.ts:4878]: There are over 100 translated strings of shortcut in this translation file. Should and how will them all be removed?

Comment (lucydodo) [reply src/translations/sqlb_zh.ts:4878]: Hmm.. Never mind. After approve this changes and merge this PR, and I will delete them. :)

**Review by lucydodo — APPROVED** [review]

**Comment by lucydodo** [comment]
Nice work. It just squash and merged. Thanks for contribution. :smile:

**Pull request merged**